test/integration/bodyFormData.spec.ts

```diff
@@ -19,48 +20,47 @@ describe("Integration tests for BodyFormData", () => {
       fileName
     );
     if (result.readableStreamBody) {
-      const buff: Buffer[] = [];
-      for await (let chunk of result.readableStreamBody) {
-        buff.push(chunk as Buffer);
-      }
-      assert.deepEqual(Buffer.concat(buff), fileContent);
+      const buff = await readStreamToBuffer(result.readableStreamBody);
+      assert.deepEqual(buff, fileContent);
     } else {
       assert.fail("ReadableStreamBody must not be null!!!");
     }
   });
 
-  it("should correctly accept file via body", async function() {
-    client = new BodyFormDataClient();
+  it.skip("should correctly accept file via body", async function() {
+    client = new BodyFormDataClient({ allowInsecureConnection: true });
     const fileName: string = `sample.png`;
     const filePath: string = `${__dirname}/../res/${fileName}`;
     const fileContent = fs.readFileSync(filePath);
     const result: FormdataUploadFileViaBodyResponse = await client.formdata.uploadFileViaBody(
       fileContent
     );
     if (result.readableStreamBody) {
-      let str: string = "";
-      for await (let chunk of result.readableStreamBody) {
-        str += chunk as string;
-      }
-      assert.deepEqual(str, fileContent.toString());
+      const buff = await readStreamToBuffer(result.readableStreamBody);
+      assert.deepEqual(buff, fileContent);
     } else {
       assert.fail("ReadableStreamBody must not be null!!!");
     }
   });
 
   it("should report upload/download progress", async function() {
+    client = new BodyFormDataClient({ allowInsecureConnection: true });
     const content = new Uint8Array(1024 * 1024 * 1);
     let uploadNotified = false;
     let downloadNotified = false;
     const response = await client.formdata.uploadFileViaBody(content, {
       requestOptions: {
         onUploadProgress: ev => {
           uploadNotified = true;
-          ev.loadedBytes.should.be.a("Number");
+          console.log(ev);
+          assert.ok(typeof ev.loadedBytes === "number");
+          //ev.loadedBytes.should.be.a("Number");

```

Comment:
I think we can remove this commented line and the one on L63

Reply:
Done